server/controllers/todosController.js

```diff
@@ -0,0 +1,160 @@
+const { Todo } = require('../models');
+
+class todosController {
+    static async getTodos (req, res) {
+        try {
+            let data = await Todo.findAll({ attributes: { exclude: ['createdAt', 'updatedAt'] } });
+
+            return res.status(200).json(data);
+        } catch (err) {
+            return res.status(500).json('Server Error');
+        }
+    }
+
+    static async postTodos (req, res) {
+        try {
+            let data = await Todo.create(req.body);
+
+            data = {
+                id: data.id,
+                title: data.title,
+                description: data.description,
+                status: data.status,
+                dueDate: data.dueDate
+            };
+
+            return res.status(201).json(data);
+        } catch (err) {
+            if (err.errors) {
+                let msg = [];
+
+                await err.errors.forEach(e => {
+                    msg.push(e.message);
+                });
+
+                return res.status(404).json({ name: 'Validation Error', msg: msg });
```

Comment:
ingat 404 itu untuk `Not Found`, kalo ini jatohnya `Bad Request` karena salah input dari client, jadi harusnya status 400